src/components/all-on-one-form-eta-button/all-on-one-form-eta-button.html

```diff
@@ -0,0 +1,7 @@
+<ion-row align-items-center no-padding [ngClass]="{'active' : isEnabled, 'pAction': pActive, 'vAction': vActive }">
```

Comment:
pAction, vAction?

Reply:
If a physical action or a verbal action took place, add class to show sudo element p or v in button. Could probably be worded better.